HuntTheWumpus/HuntTheWumpus/Hazard.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HuntTheWumpus
+{
+    public abstract class Hazard
```

Comment:
Since Bat's could just call the "MoveTo()" method to teleport the player, The hazards could be an enum

Reply:
That is true, but I don't think there is any harm in leaving them as classes, in case we want to implement any methods down the road.  Your thoughts?